webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'web-risk',
-    filename: './web-risk.js',
+    library: 'WebRiskServiceV1Beta1',
```

Comment:
So... (1) this is breaking, but (2) what we had before did not work anyway (there must be no dash in the library name), so I guess it's OK to leave it this way.

Reply:
got it, we have package name as `@google-cloud/web-risk` so I guess we may need change from the webpack side?

Reply:
cool